scripts/mkcloud

```diff
@@ -1484,11 +1506,11 @@ function sanity_checks()
             ;;
     esac
 
-    if [[ $want_sles12 = 1 ]] && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 3 ]] ; then
+    if [[ $want_sles12 = 1 ]] && [[ $want_ceph = 1 ]] && [[ $(nodes number normal) -lt 3 ]] ; then
         complain 113 "Please increase number of nodes for this setup, minimal nodenumber=3"
     fi
 
-    if iscloudver 6plus && [[ $want_ceph = 1 ]] && [[ $nodenumber -lt 3 ]] ; then
```

Comment:
This is correct, but it will be part of the next PR that I am working on.
This PR is a refactoring without changing the behaviour. The next one will.